src/index.html

```diff
@@ -11,7 +11,136 @@
+          minlength="4"
+          maxlength="16"
+          autocomplete="off"
+          >
+        </label>
+        <label for="name" class="label_field">
+          Name
+          <input type="text" 
+          id="name"
+          name="name"
+          autocomplete="off"
+          >
+        </label>
+        <label class="label_field">
+          How old are you?
+          <input type="number"
+          value="12"
+          name="age"
+          min="1"
+          max="100"
+          >
+        </label>
+        <label class="label_field">
+          Full date of birth:
+          <input
+          type="date"
+          name="date"
+          >
+        </label>
+        <label class="label_field">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="agree"
+          checked
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fielsed_field">
+        <legend>Registration</legend>
+        <label class="label_field">
+        E-mail:
+          <input name="user-email"
+        type="email"
+        placeholder="email@example.com"
+          >
+        </label>
+        <label class="label_field">
+          Password:
+          <input name="user-password" type="password">
+        </label>
+      </fieldset>
+      <fieldset class="fielsed_field">
+        <legend>An interesting fact about you?</legend>
+        <div class="label_field">
+            Do you love cats?
+          <label for="cats_yes">
+            <input type="radio" 
+            name="pets" 
+            id="cats_yes"
+            >
+            Yes
+          </label>
+          <label for="cats_no">
+            <input type="radio"
+            name="pets"
+            id="cats_no"
+            >
+            No
+          </label>
+        </div>
+        <label class="label_field">
+          What is your favorite color?
+          <input name="color" type="color">
+        </label>
+        <label class="label_field">
+          What time do yuor go to bed?
+          <input name="time" type="time">
+        </label>
+        <label class="label_field">
+          What are yuor favorite brand of cars?
+          <select name="car" multiple>
+            <option value="coffee">BMW</option>
+            <option value="tea">Audi</option>
+            <option value="juice">Lada</option>
+            <option value=" "></option>
+          </select>
+        </label>
+        <label class="label_field">
+          How do you rate our work?
+          <input name="work"
+          type="range"
+          min="4"
+          max="16"
+          step="1"
+          value="8"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fielsed_field">
+        <legend>Additional info:</legend>
+        <label class="label_field">
+        Comments:
+          <textarea name="comments"
+          cols="30"
+          rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="comments"
            cols="30"
            rows="2"
      ></textarea>
```

It should fix extra spaces in textarea field